include/rocksdb/env.h

```diff
@@ -1355,12 +1363,12 @@ class EnvWrapper : public Env {
     return target_->IncBackgroundThreadsIfNeeded(num, pri);
   }
 
-  void LowerThreadPoolIOPriority(Priority pool = LOW) override {
```

Comment:
My C++ is not good enough to determine whether it is OK to remove a default value in inherited class.

Reply:
I think as long as we do not call the method on the derived class directly, it's fine to omit the default value in the inherited class. We always call the methods through `Env*`.

I tried this snippet:
```
#include <iostream>

class Base {
 public:
  virtual void f(int x = 10) {
   std::cout << "Base" << x << std::endl;
  }
};

class Derived : public Base {
 public:
  virtual void f(int x = 1) {
   std::cout << "Derived" << x << std::endl;
  }
};


int main() {
 Derived d;
 Base *b = &d;
 b->f();
 return 0;
}
```
The output is `Derived10` instead of `Derived1`.